src/Button/Button.js

```diff
@@ -7,25 +7,27 @@ import ButtonBase from '../internal/ButtonBase';
 import { getContrastText } from '../styles/palette';
 
 export const styleSheet = createStyleSheet('Button', (theme) => {
+  const { typography, palette, transitions, shadows } = theme;
```

Comment:
That made me wondering. In many places, I have seen that we are the full path (e.g. `theme.typography.fontWeightMedium`).
I was assuming it was on purpose to *grep* more easily where theme property are used.
I guess that wasn't the case.